src/translations/en.json

```diff
@@ -838,7 +838,8 @@
               "input_number": "Reload input numbers",
               "input_datetime": "Reload input date times",
               "input_select": "Reload input selects",
-              "template": "Reload template entities"
+              "template": "Reload template entities",
+              "universal": "Reload universal entities"
```

Comment:
Universal, is that just the media player? I really don't like the naming, and I guess it is confusing with the other "universal" integrations?

<https://www.home-assistant.io/integrations/#search/universal>

Reply:
We use "generic" for other platforms I think?

home-assistant.io/integrations/#search/generic

Reply:
The naming choice is a bit odd ..... but it would be a breaking change to rename it. 🤷

Reply:
Since its only `media_player` we could make it `Reload universal media player entities`

Reply:
For reference https://github.com/home-assistant/core/issues/39033#issuecomment-676479193

Reply:
We should change our reload UI 😄 that's a long list...

Any idea what would happen if we called all reload services at the same time?

Reply:
I do like the idea that sean had for being able to click chips and reload many at once

Reply:
but if we are to do that lets get this PR in and then update the full UI in another

Reply:
It should be relatively safe to reload a few at the same time.  We are basically doing that at startup anyways.

Reply:
Would it be best if there was some way to queue them?

Reply:
```suggestion
              "universal": "Reload universal media player entities"
```